package/scripts/create-index.ts

```diff
@@ -4,13 +4,17 @@ import format from 'pg-format';
 // Relative
 import { readyToCreateOrDrop, getTextColumnsFromTable } from './lib.js';
 // Constants
-import { VECTOR_COLUMN, INDEX_PREFIX } from '../constants.js';
+import { VECTOR_COLUMN, INDEX_PREFIX } from '../src/constants.js';
```

Comment:
The 'official' way is to have `.js` at the end. Although it is very popular to drop it.

I used to prefer without it... but here I've gone the official route because...

The build script in package.json `"build:watch-swc": "swc src -w --out-dir build"`. 

It's 10x faster than `tsc` for compiling typescript, but I couldn't get it to work w/o file extensions. Here's a related issue https://github.com/swc-project/swc/issues/5346

Reply:
TIL: this very issue causes much discord in the TS world:
https://news.ycombinator.com/item?id=35752069

Reply:
> (last I checked) there isn’t even an agreed upon plan to resolve the inconsistency

Sadly, this is not suprising